lisa/transformers/hyperv_preparation.py

```diff
@@ -33,6 +33,9 @@ def _internal_run(self) -> Dict[str, Any]:
         # Enable Hyper-V
         hv = self._node.tools[HyperV]
 
+        # Configure lisa working path to use free disk.
+        hv.use_raw_disk_for_lisa_working_dir()
```

Comment:
It shouldn't be the default way. What happened, if it's running on Azure?

Reply:
This is written and tested on azure. My use case is also Azure only

Reply:
I see. Please make it optional. Only if give a path in config, then create the data disk.

Reply:
Also use_raw_disk_for_lisa_working_dir() will skip if there is not raw disk available on the Host.

Reply:
It's not flexible enough. Please create configurations in the runbook. For example,
```
data_disk:
    count: # int
    source: # new/exist/raw
    path: # for new or exist. It must be a file name for exist. It can be file name or path for new. If it ends with vhd, it means a file name, otherwise a path. If it's not set, use lisa working path for new.
```